res/themes/status/css/_status.scss

```diff
@@ -76,6 +76,7 @@ $focus-brightness: 125%;
 
 // red warning colour
 $warning-color: #F69E98;
+$warning-bg-color: #DF2A8B;
```

Comment:
That's the same bright pink as a mention, right? Not too bright as a background color?

Reply:
Nope, a mention is #ff0064 - this is a bit darker which makes it a bit less overwhelming for the background, although we're still not quite sure whether it's what @lampholder meant in the design ;)